scripts/runtests.py

```diff
@@ -0,0 +1,128 @@
+import json
+import re
+import subprocess
+import sys
+from pathlib import Path
+from typing import Optional
+
+try:
+    from termcolor import colored
+except ImportError:
+
+    def colored(text: str, color: str = "") -> str:  # type: ignore[misc]
+        return text
+
+
+# Popular Flake8 extensions not used in Typeshed that cause errors
+_STRICTER_CONFIG_FILE = "pyrightconfig.stricter.json"
+_SUCCESS = colored("Success", "green")
+_SKIPPED = colored("Skipped", "yellow")
+_FAILED = colored("Failed", "red")
+
+
+def _parse_jsonc(json_text: str) -> str:
+    # strip comments from the file
+    lines = [line for line in json_text.split("\n") if not line.strip().startswith("//")]
+    # strip trailing commas from the file
+    valid_json = re.sub(r",(\s*?[\}\]])", r"\1", "\n".join(lines))
+    return valid_json
+
+
+def _get_strict_params(stub_path: str) -> list[str]:
+    with open(_STRICTER_CONFIG_FILE) as file:
+        data = json.loads(_parse_jsonc(file.read()))
+    if stub_path in data["exclude"]:
+        return []
+    return ["-p", _STRICTER_CONFIG_FILE]
+
+
+if __name__ == "__main__":
+    try:
+        path = sys.argv[1]
+    except IndexError:
+        print("Missing path argument in format <folder>/<stub>", file=sys.stderr)
+        sys.exit(1)
+    path_tokens = Path(path).parts
+    assert len(path_tokens) == 2, "Path argument should be in format <folder>/<stub>"
+    folder, stub = path_tokens
+    assert folder in {"stdlib", "stubs"}, "Only the 'stdlib' and 'stubs' folders are supported"
+    stubtest_result: Optional[subprocess.CompletedProcess[bytes]] = None
+    pytype_result: Optional[subprocess.CompletedProcess[bytes]] = None
```

Comment:
If you add `from __future__ import annotations` at the top of the file, we can use PEP 604 here:

```suggestion
    stubtest_result: subprocess.CompletedProcess[bytes] | None = None
    pytype_result: subprocess.CompletedProcess[bytes] | None = None
```